### PR TITLE
Tan11389/100.11.2

### PR DESCRIPTION
**Pull request by tanneryould**
This PR merges changes from 100.11.2 into main. It includes 14 commits and is therefore a cleaner and probably safer commit to main than previous open PRs. It omits a few minor bug fixes and instead just includes changes mentioned in the 100.11.2 release report.

### CODE REVIEW

**Review by JamesMBallard — APPROVED** [review]
👍

**Comment by tanneryould** [comment]
The metadata check is raising a false error which was resolved by this PR https://github.com/Esri/arcgis-runtime-samples-qt/pull/1254. The test won't update unless there's another commit however.

File: ArcGISRuntimeSDKQt_QMLSamples/Layers/DisplayOgcApiFeatureCollection/DisplayOgcApiFeatureCollection.pro [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#-------------------------------------------------
+
+TEMPLATE = app
+
+# additional modules are pulled in via arcgisruntime.pri
+QT += opengl qml quick
+
+CONFIG += c++14
+
+ARCGIS_RUNTIME_VERSION = 100.12
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_QMLSamples/Layers/DisplayOgcApiFeatureCollection/DisplayOgcApiFeatureCollection.pro:23]:
```suggestion
ARCGIS_RUNTIME_VERSION = 100.11
```

File: ArcGISRuntimeSDKQt_QMLSamples/Layers/DisplayOgcApiFeatureCollection/DisplayOgcApiFeatureCollection.qml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import Esri.ArcGISRuntime 100.12
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_QMLSamples/Layers/DisplayOgcApiFeatureCollection/DisplayOgcApiFeatureCollection.qml:18]:
```suggestion
import Esri.ArcGISRuntime 100.11
```

File: ArcGISRuntimeSDKQt_QMLSamples/UtilityNetwork/CreateLoadReport/CreateLoadReport.pro [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+#-------------------------------------------------
+
+TEMPLATE = app
+
+# additional modules are pulled in via arcgisruntime.pri
+QT += opengl qml quick
+
+CONFIG += c++14
+
+ARCGIS_RUNTIME_VERSION = 100.12
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_QMLSamples/UtilityNetwork/CreateLoadReport/CreateLoadReport.pro:23]:
```suggestion
ARCGIS_RUNTIME_VERSION = 100.11
```

File: ArcGISRuntimeSDKQt_QMLSamples/UtilityNetwork/CreateLoadReport/CreateLoadReport.qml [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import Esri.ArcGISRuntime 100.12
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_QMLSamples/UtilityNetwork/CreateLoadReport/CreateLoadReport.qml:18]:
```suggestion
import Esri.ArcGISRuntime 100.11
```

**Review by ldanzinger — CHANGES_REQUESTED** [review]
couple of 100.12 -> 100.11 changes needed

**Review by ldanzinger — APPROVED** [review]

**Pull request merged**